Formula/neovim-qt.rb

```diff
@@ -0,0 +1,32 @@
+class NeovimQt < Formula
+  desc "Neovim GUI, in Qt5"
+  homepage "https://github.com/equalsraf/neovim-qt"
+  url "https://github.com/equalsraf/neovim-qt/archive/v0.2.16.1.tar.gz"
+  sha256 "971d4597b40df2756b313afe1996f07915643e8bf10efe416b64cc337e4faf2a"
+  license "ISC"
+  head "https://github.com/equalsraf/neovim-qt.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "neovim"
+  depends_on "qt@5"
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", ".."
```

Comment:
```suggestion
      system "cmake", "..", *std_cmake_args
```

Reply:
Is `std_cmake_args` documented somewhere?

Reply:
Does `std_cmake_args` include `-DCMAKE_BUILD_TYPE=Release` (or RelWithDebugInfo)?

You'll definitely want to do a `Release` build of some kind...

An easy way to verify this is `nvim-qt --versoin`:
```
$ nvim-qt --version
NVIM-QT v0.2.17.9999
Build type: Release <-- Anything other than "Debug" is fine
...
```